pkg/codegen/internal/test/testdata/output-funcs/nodejs/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+    "compilerOptions": {
+        "outDir": "bin",
+        "target": "es2016",
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "declaration": true,
+        "sourceMap": true,
+        "stripInternal": true,
+        "experimentalDecorators": true,
+        "noFallthroughCasesInSwitch": true,
+        "forceConsistentCasingInFileNames": true,
+        "strict": true
+    },
+    "files": [
+        "codegenTests.ts",
```

Comment:
I had to jump through hoops to get codegenTests.ts listed here. Could we do `**/*.ts` pattern instead? Or this is quite intentional for codegen to list files by name.